Documentation/upgrade-patch.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Patch Upgrades
+weight: 61
+indent: true
+---
+
+# 0.8 Patch Upgrades
+
+The changes in patch releases are scoped to the minimal changes necessary and are expected to be straight forward to upgrade. 
+This guide will walk you through the manual steps to upgrade the software in a Rook cluster from an 0.8 version to another patched version of 0.8. 
+For example, upgrade from `v0.8.0` to `v0.8.1`. 
+
+After each component is upgraded, it is important to verify that the cluster returns to a healthy and fully functional state. 
+
+## Considerations
+With this manual upgrade guide, there are a few notes to consider:
+* **WARNING:** Upgrading a Rook cluster is a manual process in its very early stages.  There may be unexpected issues or obstacles that damage the integrity and health of your storage cluster, including data loss.
+* This guide assumes that your Rook operator and its agents are running in the `rook-ceph-system` namespace. It also assumes that your Rook cluster is in the `rook-ceph` namespace.  If any of these components is in a different namespace, search/replace all instances of `-n rook-ceph-system` and `-n rook-ceph` in this guide with `-n <your namespace>`.
+
+## Prerequisites
+In order to successfully upgrade a Rook cluster, the following prerequisites must be met:
+* The cluster should be in a healthy state with full functionality.
+Review the [health verification section](upgrade.md#health-verification) in the main upgrade guide in order to verify your cluster is in a good starting state.
+* All pods consuming Rook storage should be created, running, and in a steady state.  No Rook persistent volumes should be in the act of being created or deleted.
+
+## Upgrade Process
+The general flow of the upgrade process will be to upgrade the version of a Rook pod, verify the pod is running with the new version, then verify that the overall cluster health is still in a good state.
+
+In this guide, we will be upgrading a live Rook cluster running `v0.8.0` to version `v0.8.1`.
+
+### Operator
+The operator controls upgrading all the components and is generally the first component to be updated.
+The operator can be updated by setting the deployment version.
+
+```bash
+kubectl -n rook-ceph-system set image deploy/rook-ceph-operator rook-ceph-operator=rook/ceph:v0.8.1
+```
+
+The operator pod will automatically be restarted by Kubernetes with the new version.
+
+Once you've verified the operator is `Running` and on the new version, verify the health of the cluster is still OK.
+Instructions for verifying cluster health can be found in the [health verification section](#health-verification).
```

Comment:
this health-verification link doesn't exist in this guide, it exists in the main guide.  link it fully like above (`[health verification section](upgrade.md#health-verification)`)